sql/connect/server/src/main/scala/org/apache/spark/sql/connect/ml/MLException.scala

```diff
@@ -36,3 +36,17 @@ private[spark] case class MLCacheInvalidException(objectName: String)
       errorClass = "CONNECT_ML.CACHE_INVALID",
       messageParameters = Map("objectName" -> objectName),
       cause = null)
+
+private[spark] case class MlItemSizeExceededException(
```

Comment:
Let's make sure the client side error mappings can deal with this new Exception. cc @itholic .